admin/spec/models/completed-checks.spec.js

```diff
@@ -24,7 +24,6 @@ describe('CompletedChecks model', () => {
     check.receivedByServerAt = undefined
     check.validate(err => {
       expect(err).toBeDefined()
-      expect(err.message).toBe('completedChecks validation failed')
```

Comment:
how come this was removed?  seems out of context 🤔

Reply:
The expectation was brittle.  That is not an error message out app produces - rather it is one produced from the mongoose library, which changed when I upgraded the mongoose package to latest.

It seems sufficient that it checks that there is an error caused by the `receivedByServerAt` field on the next line.

Reply:
makes sense to me 👍